data/json/effects_on_condition/medicine_eocs/regen_serum_eoc.json

```
@@ -0,0 +1,97 @@
+[
+  {
+    "type": "jmath_function",
+    "id": "spent_to_ingested_ratio",
+    "//": "return ratio between calories you spent yesterday and calories you ingested yesterday. if bigger than 1, you consumed more calories than you ingested",
+    "num_args": 0,
+    "return": "max( get_calories_daily('type':'spent', 'day':'1'), 1 ) / max( get_calories_daily('type':'ingested', 'day':'1'), 1 )"
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_SERUM_REGEN",
+    "effect": [
+      { "math": [ "_serum_duration", "=", "rng(time('1 d'), time('2 d'))" ] },
+      { "u_add_effect": "regen_serum_main", "duration": { "math": [ "_serum_duration" ] } },
+      { "u_add_effect": "regen_serum_crash_cardio", "duration": { "math": [ "_serum_duration * rng(1.5, 2.5)" ] } },
+      { "queue_eocs": "EOC_APPLY_REGEN_SERUM_CRASH", "time_in_future": { "context_val": "_serum_duration" } },
```

```suggestion
      { "math": [ "serum_duration", "=", "rng( 1, 2) ] },
      { "u_add_effect": "regen_serum_main", "duration": { "math": [ "serum_duration" ] } },
      { "u_add_effect": "regen_serum_crash_cardio", "duration": { "math": [ "serum_duration * rng(1.5, 2.5)" ] } },
      { "queue_eocs": "EOC_APPLY_REGEN_SERUM_CRASH", "time_in_future": { "context_val": "serum_duration" } },
```
Forgot to change this part.